src/main/java/seedu/contax/MainApp.java

```diff
@@ -45,6 +45,7 @@
     protected Storage storage;
     protected Model model;
     protected Config config;
+    private boolean isFirstRun;
```

Comment:
This flag is not used as far as I can tell.
```suggestion

```

Reply:
FIxed